### PR TITLE
[TEVA 1032] MAT dashboard page HTML preparation

### PR DESCRIPTION
**Pull request by alexbowen**
*** these screenshots show a demo, it hasnt been added to page in this PR ***
Screenshot 2020-07-08 at 12 24 32
Screenshot 2020-07-08 at 12 22 33

tweeks to dashboard page markup so it matches design and will accept new filter controls

https://dfedigital.atlassian.net/browse/TEVA-1032

### CODE REVIEW

File: app/views/hiring_staff/organisations/show.html.haml [hunk 1]
```diff
@@ -2,28 +2,25 @@
 
 .school.govuk-grid-row
   .govuk-grid-column-full
-    %h1.govuk-heading-l.mb0
+    %h1.govuk-heading-l.mb0{ class: 'govuk-!-margin-bottom-2' }
```
Comment (cpjmcquillan) [inline app/views/hiring_staff/organisations/show.html.haml:5]: we can lose the `.mb0`

**Comment by cpjmcquillan** [comment]
Can you add a screenshot of what these changes look like please?

**Comment by alexbowen** [comment]
as above, but the table will be full width as there are no filters in place yet

**Comment by alexbowen** [comment]
pointless having PR for this, will be in another

**Pull request closed without merging**